Bugzilla/Install/Localconfig.pm

```diff
@@ -297,7 +294,7 @@ sub update_localconfig {
         print colored(install_string('lc_new_vars', { localconfig => $filename,
                                                       new_vars => wrap_hard($newstuff, 70) }),
                       COLOR_ERROR), "\n";
-        exit;
+        exit unless $params->{use_defaults};
```

Comment:
So that we can get some default options when the localconfig file doesn't exist.